### PR TITLE
completed assessment

### PR DESCRIPTION
**Pull request by RoderickVongsa**
# Submitting Unit 4 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [ ] JavaScript Coding Challenges with Jest
- [ ] Interview Questions
- [ ] Ruby Challenges with RSpec

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -11,26 +11,75 @@
...
 
 const fibonacciLength2 = 10
+it("should generate Fibonacci sequence of length 10", () => {
+    const result = fibonacci(fibonacciLength2);
+    expect(result).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55]);
+  });
+});
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
```
Comment (SunkissedQueen) [inline code-challenges.test.js:46]: A bit hard to follow the logic. I had to look over it a couple of times. I think that a refactor would come in handy. It is great that you accomplished a passing test. I would say go back and see how you can create the test with less lines. If you need help, please reach out.

**Review by SunkissedQueen — APPROVED** [review]
Rod, you are approved to push your work. I do encourage you go back for a refactor on your code. For these jest tests, one `it` method can cover the test. However, you can use the expect statements to cover the edge cases.  If you need help, please reach out. Please review these following section:
- [Jest](https://github.com/learn-academy-2023-india/syllabus/blob/main/javascript/jest.md)
Please reach out if you need any help with troubleshooting and creating functions. Keep striving. This work is currently on the high end of advanced beginner. On your next assessment, please also begin thinking about more content that you can add to your interview answers.

File: code-challenges.test.js (hunk 1 above)
Comment (RoderickVongsa) [reply code-challenges.test.js:46]: Thanks Charlean, question am I allowed to merge my assessment or should I go back and change anything?
